src/IO/lbsLoadReaderVT.py

```diff
@@ -169,13 +174,22 @@ def read(self, node_id, time_step=-1, comm=False):
                             print(bcolors.HEADER
                                 + "[LoadReaderVT] "
                                 + bcolors.END
-                                + "iteration = {}, object id = {}, time = {}".format(
+                                + "iteration = {}, object id = {}, time = {}, subphases = {}".format(
                                 phase,
                                 o_id,
-                                time))
+                                time,
+                                Nsubphases,
+                                end=''))
+                                
+                            for i in range(0,Nsubphases):
+                                print("subphase-time-" + str(i) + "= {}".format(
+                                    row[4 + i]
+                                end='')
```

Comment:
If we had a parsing error, then we can't trust that the value of `Nsubphases` accurately represents the content of the row we failed to parse.

Reply:
A parsing error? As in the CSV reader fails?

Reply:
The parsing errors that could be caught by this `except` clause would be when the values within the row weren't suitable for conversion to `int` or `float`. If that happens, some of the variable assignments would have failed, and so our exception handling would raise new exceptions on accesses to undefined variables. Worse, if the failure is not on the first row, by Python's variable scoping rules, those variables *will* be initialized, with the values from previous loop iterations, and the error message will be horribly confused.